src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -314,32 +316,6 @@ class AppleDeviceInterface extends InterfacePrototype {
             poll()
         })
     }
-    /**
```

Comment:
I've moved this to `InterfacePrototype` then the interfaces pass them to the `controllers` it's all rather weird as I mention in my previous comment.